src/node_dotenv.cc

```diff
@@ -130,50 +135,81 @@ void Dotenv::ParseContent(const std::string_view input) {
 
   while (!content.empty()) {
     // Skip empty lines and comments
+    // Example:
+    //   # This is a comment
+    //
+    //   KEY=value
     if (content.front() == '\n' || content.front() == '#') {
       auto newline = content.find('\n');
       if (newline != std::string_view::npos) {
         content.remove_prefix(newline + 1);
+        // Trim spaces after skipping comments or empty lines to handle
+        // cases where there might be trailing whitespace
+        content = trim_spaces(content);
         continue;
+      } else {
+        // If no newline is found, we've reached the end of content
```

Comment:
I assume this line gets called for `#ABC=1` input. Can you add this example here?